web-common/src/components/chip/removable-list-chip/RemovableListChip.svelte

```diff
@@ -77,7 +77,7 @@ are details left to the consumer of the component; this component should remain
         {active}
       />
     </Chip>
-    <div slot="tooltip-content" transition:fly|local={{ duration: 100, y: 4 }}>
+    <div slot="tooltip-content" transition:fly={{ duration: 100, y: 4 }}>
```

Comment:
`|local` was added to fix issues with elements lingering. Has it been fixed in svelte 4? Applies to all instances of this change.

Reply:
See above – transitions are now local by default